docs/shadow_config_spec.md

```diff
@@ -1,8 +1,9 @@
 # Shadow Configuration Specification
 
-Shadow uses the standard YAML format to accept configuration options from users.
-The following describes Shadow's YAML format and all of the options that Shadow
-supports that can be used to customize a simulation.
+Shadow uses the standard [YAML 1.2](https://yaml.org/spec/1.2.2/) format to
+accept configuration options from users.  The following describes Shadow's YAML
+format and all of the options that Shadow supports that can be used to customize
+a simulation.
```

Comment:
nit: I've always wanted to remove the "from users" text. Since the person reading this is probably a user, this text seems weird.